tests/integration/conftest.py

```diff
@@ -162,7 +173,10 @@ def _batch_setup_for_datasource(
     request: pytest.FixtureRequest,
     _cached_test_configs: dict[TestConfig, BatchTestSetup],
     _cached_secondary_test_configs: dict[UUID, BatchTestSetup],
-    _cleanup,
+    session_sql_engine_manager: SessionSQLEngineManager,
```

Comment:
This is the 1 place the new fixture is used. I had set logging in an early version of this PR so you can see engine creation and cache hits: https://github.com/great-expectations/great_expectations/actions/runs/15477818453?pr=11228